editor/src/components/editor/editor-component.tsx

```diff
@@ -143,8 +143,8 @@ export const EditorComponentInner = React.memo((props: EditorProps) => {
         const key = Keyboard.keyCharacterForCode(event.keyCode)
 
         // TODO: maybe we should not whitelist keys, just check if Keyboard.keyIsModifer(key) is false
+        const existingInteractionSession = editorStoreRef.current.editor.canvas.interactionSession
```

Comment:
I think we should probably take the approach suggested in the above `TODO`, i.e. if `Keyboard.keyIsModifier(key)` we only update existing interactions, whereas other keys can create a new interaction

Reply:
the todo is not _quite_ about that, but I think I should flip the if-else order because this is risking a regression if we extend the whitelist in Keyboard.keyIsInteraction